### PR TITLE
[DOCS] Enables mathematical formulate in DFA chapter and adds markup to a formula

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR enables mathematical formulae in the data frame analytics chapter of the ML book and adds the required markup to the formula in the classification conceptual docs to render it correctly.

AsciiMath can be enabled by adding `:stem:` to the index.asciidoc of the chapter/book.

```
= Section title
:stem:
```
Use the following notation for the formulae:

```
stem:[sum_(i=1)^n i^3=((n(n+1))/2)^2]
```
For a centered block:

```
[stem]
++++
x/x={(1,if x!=0),(text{undefined},if x=0):}
++++
``` 

### Preview

[Classification](http://stack-docs_1091.docs-preview.app.elstc.co/guide/en/machine-learning/master/dfa-classification.html#dfa-classification-class-score)

### CODE REVIEW

File: docs/en/stack/ml/df-analytics/dfa-classification.asciidoc [hunk 1]
```diff
@@ -131,10 +131,12 @@ so it can be less than or greater than 0.5. For example, suppose our two classes
 are denoted `class 0` and `class 1`, then the value of `class_score` is always 
 non-negative and its definition is:
 
-```
-class_score(class 0) = 0.5 / (1.0 - k) * probability(class 0)
-class_score(class 1) = 0.5 / k * probability(class 1)
-```
+
+`class_score (class 0)` 
+stem:[= 0.5 / (1.0 - k) * probability(class 0)]
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/dfa-classification.asciidoc:136]: Should we move the first bit into the equation too, so it's got the same formatting?
```suggestion
stem:[\text{class_score(class 0)} = 0.5 / (1.0 - k) * probability(class 0)]
```

File: docs/en/stack/ml/df-analytics/dfa-classification.asciidoc [hunk 2]
```diff
@@ -131,10 +131,12 @@ so it can be less than or greater than 0.5. For example, suppose our two classes
 are denoted `class 0` and `class 1`, then the value of `class_score` is always 
 non-negative and its definition is:
 
-```
-class_score(class 0) = 0.5 / (1.0 - k) * probability(class 0)
-class_score(class 1) = 0.5 / k * probability(class 1)
-```
+
+`class_score (class 0)`
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/dfa-classification.asciidoc:135]:
```suggestion
```

File: docs/en/stack/ml/df-analytics/dfa-classification.asciidoc [hunk 3]
```diff
@@ -131,10 +131,12 @@ so it can be less than or greater than 0.5. For example, suppose our two classes
...
-```
-class_score(class 0) = 0.5 / (1.0 - k) * probability(class 0)
-class_score(class 1) = 0.5 / k * probability(class 1)
-```
+
+`class_score (class 0)` 
+stem:[\text{class_score(class 0)} = 0.5 / (1.0 - k) * probability(class 0)]
+
+`class\_score (class 1)` 
+stem:[= 0.5 / k * probability(class 1)]
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/dfa-classification.asciidoc:139]:
```suggestion
stem:[[\text{class_score(class 1)}= 0.5 / k * probability(class 1)]
```

File: docs/en/stack/ml/df-analytics/dfa-classification.asciidoc [hunk 4]
```diff
@@ -131,10 +131,12 @@ so it can be less than or greater than 0.5. For example, suppose our two classes
...
 
-```
-class_score(class 0) = 0.5 / (1.0 - k) * probability(class 0)
-class_score(class 1) = 0.5 / k * probability(class 1)
-```
+
+`class_score (class 0)` 
+stem:[\text{class_score(class 0)} = 0.5 / (1.0 - k) * probability(class 0)]
+
+`class\_score (class 1)`
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/dfa-classification.asciidoc:138]:
```suggestion
```

File: docs/en/stack/ml/df-analytics/dfa-classification.asciidoc [hunk 5]
```diff
@@ -131,10 +131,10 @@ so it can be less than or greater than 0.5. For example, suppose our two classes
...
 non-negative and its definition is:
 
-```
-class_score(class 0) = 0.5 / (1.0 - k) * probability(class 0)
-class_score(class 1) = 0.5 / k * probability(class 1)
-```
+
+stem:[\text{class_score(class 0)} = 0.5 / (1.0 - k) * probability(class 0)]
+
+stem:[[\text{class_score(class 1)}= 0.5 / k * probability(class 1)]
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/dfa-classification.asciidoc:137]:
```suggestion
stem:[\text{class_score(class 1)}= 0.5 / k * probability(class 1)]
```

**Comment by szabosteve** [comment]
**Backport commits:**
7.x: 7bc600e
7.8: e0cbd3c
7.7: 43633de

**Pull request merged**